src/integrations/admin/crawl-settings-integration.php

```diff
@@ -94,18 +123,11 @@ public function add_crawl_settings_tab( $dashboard_tabs ) {
 	public function add_crawl_settings_tab_content( $yform ) {
 		$this->display_premium_upsell_btn();
 
-		$yform->toggle_switch(
-			'remove_feed_post_comments_free',
-			[
-				'off' => __( 'Keep', 'wordpress-seo' ),
-				'on'  => __( 'Remove', 'wordpress-seo' ),
-			],
-			__( 'Post comment feeds', 'wordpress-seo' ),
-			'',
-			[
-				'disabled' => true,
-			]
-		);
+		echo '<h3 class="yoast-feed-crawl-settings">';
```

Comment:
can you use a class like `yoast-feed-crawl-settings-free` so opacity is set to 0.5 only in Free and not in Premium?